dev-lang/golang/golang-1.4.3.recipe

```diff
@@ -0,0 +1,73 @@
+SUMMARY="The Go Programming Language"
+DESCRIPTION="The Go Programming Language is an open source project to make \
+programmers more productive.
+
+Go is expressive, concise, clean, and efficient. Its concurrency \
+mechanisms make it easy to write programs that get the most out of multicore \
+and networked machines, while its novel type system enables flexible and \
+modular program construction. Go compiles quickly to machine code yet has the \
+convenience of garbage collection and the power of run-time reflection. It's a \
+fast, statically typed, compiled language that feels like a dynamically typed, \
+interpreted language."
+HOMEPAGE="https://golang.org/go"
+COPYRIGHT="2009-2018 The Go Authors"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://dl.google.com/go/go$portVersion.src.tar.gz"
+CHECKSUM_SHA256="9947fc705b0b841b5938c48b22dc33e9647ec0752bae66e50278df4f23f64959"
+SOURCE_DIR="go"
+PATCHES="golang-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 !x86 ?x86_64"
+SECONDARY_ARCHITECTURES="!x86"
+
+PROVIDES="
+	golang$secondaryArchSuffix = $portVersion
+	cmd:go$secondaryArchSuffix = $portVersion
+	cmd:gofmt$secondaryArchSuffix = $portVersion
+	cmd:godoc$secondaryArchSuffix = $portVersion
+	"
+if [ -n "$secondaryArchSuffix" ]; then
+	PROVIDES="$PROVIDES
+	cmd:go = $portVersion
+	cmd:gofmt = $portVersion
+	cmd:godoc = $portVersion
+	"
+fi
+
```

Comment:
No empty line here, ps, if this doesn't build for x86_gcc2 couldn't you use commandSuffix?

Reply:
I think we can bend the rules a little bit regarding the empty line. That's because we have a `if .. then .. fi` just before the `REQUIRES`.
So, imho, the empty line might be ok here. @Begasus, what do you think?

The only change I'd recommend is:
```diff
 if [ -n "$secondaryArchSuffix" ]; then
-	PROVIDES="$PROVIDES
+PROVIDES="$PROVIDES
 	cmd:go = $portVersion
 	cmd:gofmt = $portVersion
 	cmd:godoc = $portVersion
 	"
 fi
```

Reply:
Just my 2 cents, is using $secondaryArchSuffix needed? If so then it's ok by me